pkg/propertychecker/azure/checker.go

```diff
@@ -149,7 +153,9 @@ func validateCapacity(value string, operator placementv1beta1.PropertySelectorOp
 
 	capacity := uint32(valueUint) // capacity is >= 0 since it's parsed as uint.
 
-	if operator == placementv1beta1.PropertySelectorGreaterThanOrEqualTo && capacity > 0 {
+	// Adjust capacity based on operator semantics for VMSizeRecommender API.
+	// If operator is GreaterThanOrEqualTo, decrement capacity by 1 as the API checks for strictly greater than.
+	if operator == placementv1beta1.PropertySelectorGreaterThanOrEqualTo && capacity > minVMInstanceCapacity {
```

Comment:
Based on the capacity team's response, we can treat the API semantics, requested capacity in the capacity API as GE.

if cx requests >= 1 in the property, we set the 1 in the request, and capacity team is looking for >1 nodes, which is internal implementation details and could be changed.
if cx request > 1 in the property, we set the 2 in the request.